server/common/utils/helpers.js

```diff
@@ -19,6 +19,18 @@ const checkIfArrayContainsUserId = (idsArray, userId) => {
   return arrayOfStrings.indexOf(userIdAsString) !== -1;
 };
 
+const isOwner = (idsArray, userId) =>
```

Comment:
please change so that method gets cohort or list. Then we can use:
`isOwner(cohort, userId)`